pkg/app/app_reconcile_test.go

```diff
@@ -20,14 +22,54 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var noopMetrics = &metrics.AppMetrics{
```

Comment:
i dont think you have to reconstruct AppMetrics here. can just use NewAppMetrics() -- just dont register them.

Reply:
ficed